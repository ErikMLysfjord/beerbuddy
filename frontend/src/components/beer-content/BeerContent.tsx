const BeerContent = () => (
  <>
    <p
      style={{
        fontSize: "20px",
        fontWeight: "lighter",
        marginBottom: "-15px",
      }}
    >
      Nico Freccia Breweries (Beer-brand)
    </p>
    {/* name */}
    <p
      style={{
        fontSize: "30px",
        fontWeight: "medium",
        margin: "0px",
      }}
    >
      21st Amendment Bitter American (Beer-name)
    </p>

    <div style={{ display: "flex", gap: "0.2rem", alignItems: "end" }}>
      <p
        style={{
          fontSize: "30px",
          color: "#FFCC48",
          marginBottom: "0px",
        }}
      >
        137
      </p>
      <p
        style={{
          fontSize: "30px",
          marginBottom: "0px",
        }}
      >
        rated
      </p>
    </div>

    {/* small text under, "based on 547 reviews" */}
    <p
      style={{
        fontSize: "15px",
        fontWeight: "lighter",
        marginTop: "-10px",
      }}
    >
      Based on 547 reviews
    </p>

    {/* Info */}
    <div
      style={{
        display: "flex",
        gap: "1rem",
      }}
    >
      <div
        style={{
          backgroundColor: "#333333",
          padding: "10px",
          width: "80px",
          borderRadius: "10px",
          textAlign: "center",
        }}
      >
        <p style={{ margin: "0px" }}>Style</p>
        <h1 style={{ margin: "0px" }}>APA</h1>
      </div>

      <div
        style={{
          backgroundColor: "#333333",
          padding: "10px",
          width: "80px",
          borderRadius: "10px",
          textAlign: "center",
        }}
      >
        <p style={{ margin: "0px" }}>IBU</p>
        <h1 style={{ margin: "0px" }}>23</h1>
      </div>

      <div
        style={{
          backgroundColor: "#333333",
          padding: "10px",
          width: "80px",
          borderRadius: "10px",
          textAlign: "center",
        }}
      >
        <p style={{ margin: "0px" }}>Abv</p>
        <h1 style={{ margin: "0px" }}>5,4%</h1>
      </div>

      <div
        style={{
          backgroundColor: "#333333",
          padding: "10px",
          width: "80px",
          borderRadius: "10px",
          textAlign: "center",
        }}
      >
        <p style={{ margin: "0px" }}>Volume</p>
        <h1 style={{ margin: "0px" }}>4oz</h1>
      </div>
    </div>
  </>
);

export default BeerContent;
