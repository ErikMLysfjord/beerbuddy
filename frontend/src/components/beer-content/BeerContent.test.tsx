import { it, expect, describe } from "vitest";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import BeerContent from "./BeerContent";

describe("BeerContent", () => {
  it("renders correctly", () => {
    const { container } = render(<BeerContent />);
    expect(container).toMatchSnapshot();
  });
});
