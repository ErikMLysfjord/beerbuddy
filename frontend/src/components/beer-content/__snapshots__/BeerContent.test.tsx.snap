// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BeerContent > renders correctly 1`] = `
<div>
  <p
    class="_breweryName_a0d0d4"
  >
    Nico Freccia Breweries (Beer-brand)
  </p>
  <p
    class="_beerName_a0d0d4"
  >
    21st Amendment Bitter American (Beer-name)
  </p>
  <div
    class="_rating_a0d0d4"
  >
    <p
      class="_ratingNumber_a0d0d4"
    >
      137
    </p>
    <p
      class="_ratingText_a0d0d4"
    >
      rated
    </p>
  </div>
  <p
    class="_basedOn_a0d0d4"
  >
    Based on 547 reviews
  </p>
  <div
    class="_info_a0d0d4"
  >
    <div
      class="_attributeContainer_a0d0d4"
    >
      <p
        class="_attributeText_a0d0d4"
      >
        Style
      </p>
      <h1
        class="_attributeText_a0d0d4"
      >
        APA
      </h1>
    </div>
    <div
      class="_attributeContainer_a0d0d4"
    >
      <p
        class="_attributeText_a0d0d4"
      >
        IBU
      </p>
      <h1
        class="_attributeText_a0d0d4"
      >
        23
      </h1>
    </div>
    <div
      class="_attributeContainer_a0d0d4"
    >
      <p
        class="_attributeText_a0d0d4"
      >
        Abv
      </p>
      <h1
        class="_attributeText_a0d0d4"
      >
        5,4%
      </h1>
    </div>
    <div
      class="_attributeContainer_a0d0d4"
    >
      <p
        class="_attributeText_a0d0d4"
      >
        Volume
      </p>
      <h1
        class="_attributeText_a0d0d4"
      >
        4oz
      </h1>
    </div>
  </div>
</div>
`;
